modules/Story/Embargo/Embargo.tsx

```diff
@@ -20,19 +22,25 @@ function Embargo({ story }: Props) {
                     date: (
                         <>
                             <FormattedDate
-                                value={new Date(story.published_at)}
+                                value={date}
                                 year="numeric"
                                 month="long"
                                 day="numeric"
                                 timeZone={timezone}
                             />{' '}
                             <FormattedTime
-                                value={new Date(story.published_at)}
+                                value={date}
                                 hour="2-digit"
                                 minute="2-digit"
-                                timeZoneName="short"
                                 timeZone={timezone}
-                            />
+                            />{' '}
+                            UTC
+                            {date
+                                .toLocaleString(Culture.isoCode(story.culture.code), {
+                                    timeZoneName: 'longOffset',
+                                    timeZone: timezone,
+                                })
+                                .replace(/^.*? GMT/, '')}
```

Comment:
This will produce a different result compared to https://github.com/prezly/theme-kit-js/pull/634, right? Is this not a mistake?